zgrab_schema.py

```diff
@@ -202,7 +202,8 @@
     }),
     "session_ticket":SubRecord({
         "value":Binary(),
-        "length":Integer()
+        "length":Integer(),
```

Comment:
Was requested by @zakird due to ``it's a pain to calculate that on the fly''. Length is not that of base64 string, but of the ticket encoded in base64.